apps/allocations/app/components/Panel/NewAllocation.js

```diff
@@ -133,12 +133,13 @@ class NewAllocation extends React.Component {
       budgetValue,
       descriptionValue,
       amountValue,
-      recipients
+      recipients,
+      tokenValue
     } = this.state
     const allocation = {
       budgetId: this.props.budgetId,
       budgetName: budgetValue,
-      balance: amountValue * 10e17,
+      balance: BigNumber(amountValue).times(BigNumber(10).pow(tokenValue.decimals)).toString(10),
```

Comment:
This looks good, long term I think this could be a helper function something like `displayBNToString(value, decimals)`